twg/devour
# Added .npmignore file to exclude src and test directories

## Priority
Low

## What Changed & Why
We don't need to include the src or test directories in the npm bundle since the code is precompiled into the lib folder.

## Testing
- Pull branch locally
- Run `npm pack`
- Confirm that the lib folder and necessary accompanying files are included (README.md, etc)
